tests/roles/control_plane_rollback/tasks/main.yaml

```diff
@@ -42,6 +42,8 @@
                      "tripleo_ceilometer_agent_compute.service"
                      "tripleo_ceilometer_agent_ipmi.service"
                      "tripleo_ceilometer_agent_notification.service"
+                     "tripleo_ovn_cluster_north_db_server.service"
```

Comment:
Can you please also add this to the rollback docs? https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/docs_user/modules/proc_rolling-back-the-control-plane-adoption.adoc?plain=1#L65 

Otherwise LGTM :+1: